conda-forge.yml

```diff
@@ -4,3 +4,4 @@ github:
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+conda_build_tool: mambabuild
```

Comment:
We have a new `conda-smithy` release that has this a the default for now.

Reply:
Thanks for tracking this down